guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -1,40 +1,71 @@
-endif::[]
-ifndef::orcharhino[]
-** If your {Project} is connected, use the {RHCloud} to create the manifest.
-ifdef::satellite[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html-single/creating_and_managing_manifests_for_a_connected_satellite_server/index[Creating and managing manifests for a connected {ProjectServer}] in _Subscription Central_.
-endif::[]
-** If your {Project} is disconnected, use the Red Hat Customer Portal to create the manifest.
-ifdef::satellite[]
-For more information, see {RHDocsBaseURL}subscription_central/1-latest/html/getting_started_with_rhel_system_registration/adv-reg-rhel-using-rhsm_#using_manifests_con[Using manifests for a disconnected Satellite Server] in _Subscription Central_.
-endif::[]
+.Subscriptions service capabilities
+* Allows customers to view subscription usage.
+* When configured, it displays usage data from both connected and disconnected environments.
+* Tracks {Team} product usage and capacity at the fleet or account level.
+* Generates a daily snapshot of usage data across all tracked systems.
+* Displays data in a filterable dashboard at cloud.redhat.com.
+* Collects historical data for analytics, purchasing decisions, capacity planning, and risk mitigation.
+* Supports the transition to Simple Content Access (SCA), which enforces subscriptions across the organization rather than on individual systems.
+
+.Configuring your system to use the Subscriptions service
+
+Connected (Non-{Project}) users::
+As a connected, non-{Project} user, no further action needs to be taken to use the subscriptions service. 
+To track your subscription usage, log in to the Hybrid Cloud Console at https://console.redhat.com/subscriptions/usage/rhel. 
+Registering a system through the guidance in https://docs.redhat.com/en/documentation/subscription_central/1-latest/html-single/getting_started_with_rhel_system_registration/index[Getting Started with RHEL System Registration] is sufficient to ensure the system is properly counted. 
+It is recommended to set System Purpose attributes to define SLA and USAGE.
+This improves filtering and enhances reporting in the Subscriptions service.
+For more information, see {RHELDocsBaseURL}9/html/automatically_installing_rhel/proc_configuring-system-purpose-using-the-subscription-manager-command-line-tool_rhel-installer[Configuring System Purpose using the subscription-manager command-line tool].
+
+ifndef::satellite[]
+Connected {Project} Users::
+Connected {Project} Users should enable the `foreman_rh_cloud` plugin if it has not been enabled already (https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_the_subscriptions_service/proc-installing-satellite-inventory-upload-plugin_assembly-setting-up-subscriptionwatch-ctxt). The foreman_rh_cloud plugin shares inventory with the Hybrid Cloud Console enabling the usage of the subscriptions service to track usage information.
```

Comment:
I'm pretty sure foreman_rh_cloud has been enabled by default in Satellite for quite some time. Should double-check this and, if it's the case, we would not need to tell users to enable it. Rather, one just needs to configure it to one's preferences.